package.json

```diff
@@ -26,6 +26,7 @@
     "sync": "carbon-cli sync",
     "test": "cross-env BABEL_ENV=test jest",
     "test:e2e": "cross-env BABEL_ENV=test jest -c jest.e2e.config.js",
+    "typecheck": "tsc -b",
```

Comment:
This script name implies type-checking and not necessarily building. Would it make more sense for this to be `tsc --noEmit`?